webui/react/src/pages/F_ExpList/glide-table/GlideTable.tsx

```diff
@@ -274,20 +306,27 @@ export const GlideTable: React.FC<Props> = ({
         theme={theme}
         verticalBorder={verticalBorder}
         width="100%"
+        onCellClicked={handleGridSelectionChange}
+        onCellContextMenu={onCellContextMenu}
         onColumnMoved={onColumnMoved}
         onColumnResize={onColumnResize}
         onColumnResizeEnd={onColumnResizeEnd}
-        onGridSelectionChange={handleGridSelectionChange}
         onHeaderClicked={onHeaderClicked}
         onVisibleRegionChanged={handleScroll}
         //
         // these might come in handy
-        // onCellClicked={onCellClicked}
         // onItemHovered={onItemHovered}
         // onHeaderContextMenu={onHeaderContextMenu}
         // onCellContextMenu={onCellContextMenu}
       />
       <TableActionMenu {...menuProps} open={menuIsOpen} />
+      {contextMenuProps?.experiment && (
```

Comment:
can we do something a bit more explicit? for example, having `contextMenuProps : TableContextMenuProps | null` and then `contextMenuProps && (...`